src/components/modal/ModalItem.js

```diff
@@ -0,0 +1,60 @@
+import "./modalItem.scss"
+import Select from "react-select";
+
+const ModalItem = (props) => {
```

Comment:
this is good but my personal preference is to destruct items from props and than I don't need to type props each time. But this is also ok

Reply:
this component shall be deleted when refactoring starts, but I'll keep that in mind